contracts/src/v0.8/dev/automation/2_1/KeeperRegistryLogicA2_1.sol

```diff
@@ -6,19 +6,12 @@ import "./KeeperRegistryLogicB2_1.sol";
 import "./Chainable.sol";
 import {AutomationForwarder} from "./AutomationForwarder.sol";
 import "../../../interfaces/automation/UpkeepTranscoderInterfaceV2.sol";
-
-import "../../../interfaces/automation/MigratableKeeperRegistryInterface.sol";
 import "../../../interfaces/automation/MigratableKeeperRegistryInterfaceV2.sol";
 
 /**
  * @notice Logic contract, works in tandem with KeeperRegistry as a proxy
  */
-contract KeeperRegistryLogicA2_1 is
-  KeeperRegistryBase2_1,
-  Chainable,
-  MigratableKeeperRegistryInterface,
-  MigratableKeeperRegistryInterfaceV2
```

Comment:
remove direct interface enforcement - instead we use a ts test now to ensure the interface is satisfied